rotkehlchen/globaldb/upgrades/v2_v3.py

```diff
@@ -0,0 +1,361 @@
+import logging
+import sqlite3
+from collections import deque
+from typing import Deque, Dict, Tuple, Any, List
+import random
+
+# from eth_utils import to_checksum_address
+# import requests
+
+from rotkehlchen.constants.resolver import (
+    CHAINID,
+    EvmTokenKind,
+    VALID_EVM_CHAINS,
+    EVM_CHAINS_TO_DATABASE,
+    evm_address_to_identifier,
+    translate_old_format_to_new,
+)
+from rotkehlchen.globaldb.schema import (
+    DB_V3_CREATE_COMMON_ASSETS_DETAILS,
+    DB_V3_CREATE_ASSETS,
+    DB_V3_CREATE_EVM_TOKENS,
+    DB_V3_CREATE_ASSETS_EVM_TOKENS,
+    DB_V3_CREATE_MULTIASSETS,
+    DB_CREATE_ETHEREUM_TOKENS_LIST,
+    DB_CREATE_USER_OWNED_ASSETS,
+)
+
+log = logging.getLogger(__name__)
+
+
+COMMON_ASSETS_INSERT = (
```

Comment:
Here and below you are encircling a `"""` quoted string in parentheses. This is not needed. Remove them